coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -1676,6 +1676,12 @@ def view(context, node):
     x = inputs[0]
     shape = inputs[1]
 
+    if np.prod(shape.shape) == 0:
+        # Reshape to empty shape (works only for scalar) is a no op
+        assert np.prod(x.shape) <= 1, "Reshape to empty shape works only for scalar"
```

Comment:
can we just do `shape.shape == ()`?
same for `x.shape == ()`

Reply:
I can change `shape.shape == ()`

But for `x.shape`, we will also need to cover
* `x.shape == (1,)`
* `x.shape == (1, 1)`
* ...

Reply:
Sounds good!

Reply:
`assert np.prod(x.shape) <= 1, "Reshape to empty shape works only for scalar"` then this msg should be modified then?

Reply:
Clarified the error message to
```
Reshape to empty shape works only for scalar and single-element tensor
```